vllm/compilation/pass_manager.py

```diff
@@ -88,27 +88,30 @@ def __call__(self, graph: fx.Graph):
 
     def configure(self, config: VllmConfig):
         self.pass_config = config.compilation_config.pass_config
-        if self.pass_config.enable_noop:
-            self.passes += [NoOpEliminationPass(config)]
 
-        if self.pass_config.enable_sequence_parallelism:
-            self.passes += [SequenceParallelismPass(config)]
-            if self.pass_config.enable_async_tp:
-                self.passes += [AsyncTPPass(config)]
+        # Set the current vllm config to allow tracing CustomOp instances
+        with set_current_vllm_config(config, check_compile=False):
+            if self.pass_config.enable_noop:
+                self.passes += [NoOpEliminationPass(config)]
 
-        if self.pass_config.enable_fi_allreduce_fusion:
-            self.passes += [AllReduceFusionPass(config)]
+            if self.pass_config.enable_sequence_parallelism:
```

Comment:
Probably, we can add warnings in `PassConfig.__post_init__` when conflicting fusions (SP vs AllReduce+RMSNorm or AllReduce+RMSNorm+Quant vs RMSNorm+Quant) are enabled specifying which fusion will actually be applied